pkg/client/src/app/Constants.ts

```diff
@@ -22,11 +22,15 @@ export interface IEnvVars {
   UI_INGRESS_PROXY_BODY_SIZE: string;
   PROFILE: string;
 }
+
+export enum BrandType {
+  Konveyor = "konveyor",
+  MTA = "mta",
+}
 export const ENV: IEnvVars = window["_env"] || {};
 export const isAuthRequired = ENV.AUTH_REQUIRED !== "false";
 export const uploadLimit = ENV.UI_INGRESS_PROXY_BODY_SIZE || "500m";
-export const isPropietaryAllowed =
-  ENV.PROFILE === "konveyor" || ENV.PROFILE === "" ? true : false;
+export const APP_BRAND = ENV.PROFILE as BrandType;
```

Comment:
@ibolton336 maybe on 2.1.2 we should have this be:

```ts
export const APP_BRAND = (ENV.PROFILE || BrandType.Konveyor) as BrandType;
```

Just because below you have places where if PROFILE isn't set you'll end up with Red Hat branding, which is probably not ideal for someone building the upstream app

Reply:
Good call - I'll make the change.